spikeinterface_gui/curationview.py

```diff
@@ -302,10 +325,56 @@ def _panel_get_merge_table_row(self):
         else:
             return selected_items
 
-
-
-CurationView._gui_help_txt = """Curation includes potential delete + merge
-Must export or apply to analyzer to make persistent
-Click on items to make then visible
-Right click to remove merges or restore deletions
+    def _panel_handle_shortcut(self, event):
+        if event.data == "restore":
+            self.restore_units()
+        elif event.data == "unmerge":
+            self.unmerge_groups()
+
+    def _panel_submit_to_parent(self, event):
```

Comment:
This is specific for your lab.
We need a proper way to inject this function externally.
Some option should be offered in the main function to propagate this and a button name.

Reply:
I think it could be a common use case in panel to embed in a parent app. It would also make it very hard to make it customizable. Let's leave it like this for now